lib/rules/no-eol-whitespace/index.js

```diff
@@ -179,6 +211,14 @@ const rule = function(on, options, context) {
         },
         isRootFirst
       );
+
+      const lastEOL = root.raws.after.lastIndexOf("\n");
```

Comment:
What about `\r`? It is rare use case, but some old os use `\r` as end of line

Reply:
done